### PR TITLE
Nitpick: Designer Edits Pt 3

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
This PR contains edits requested in this [excel](https://docs.google.com/spreadsheets/d/1-uzCCu48XLRHw-uEnCEO1hyksYOItZXokiCBcxajltI/edit?pli=1#gid=0)

Please test:
- Newsletter confirmation popup: center on the screen mobile
- Footer: Animated hover and center 
- Newsletter Sign up: Center text, and condense space more 
- Archive Sorting: Animated bold on hover
- Archive Preview: Lower brightness on hover and switch arrow svg
- About/ Resource button: Animate fade in color
- Privacy Policy: Remove x axis scroll bar (@jzhzhang66 please check on your computer)
- Page Description: Increase width of image to span more of the screen
- Box Link: Increase left margin, for text description on mobile

### CODE REVIEW

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Should be bolded after click~
image

File: components/footer/Footer.style.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ const FooterContainer = styled.div`
   width: 100%;
```
Comment (jean-zhang) [inline components/footer/Footer.style.js:7]: Unrelated nit: Footer links are not working

File: components/oval-button/OvalButton.style.js [hunk 2]
```diff
@@ -14,6 +14,11 @@ const Button = styled.a.attrs({
   width: clamp(7.5em, 10vw, 8em);
   height: 3.8em;
 
+  transition: background-color 0.3s linear;
+  -moz-transition: background-color 0.3s linear;
```
Comment (jean-zhang) [inline components/oval-button/OvalButton.style.js]: it seems transition is now supported for all browsers except opera so we can just use `transition` (I'd double check though)

**Review by jean-zhang — APPROVED** [review]

File: components/footer/Footer.style.js (hunk 1 above)
Comment (isabellberzsenyi) [reply components/footer/Footer.style.js:7]: fixed

**Review by jzhzhang66 — APPROVED** [review]

**Pull request merged**